### PR TITLE
suite-nonsplit-product-measurement-units

### PR DESCRIPTION
**Pull request by gorav-sharma-metacube**
## PR Description
CC-16826: suite-nonsplit-product-measurement-units

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Test Setup    TestSetup
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+Get_product_measurement_unit_by_id
+    When I send a GET request:    /product-measurement-units/METR
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot:11]: Please use a variable here, not a hardcoded value

Comment (gorav-sharma-metacube) [reply tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot]: fixed

**Comment by gorav-sharma-metacube** [comment]
fixed

File: tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+Get_product_measurement_unit_by_id
+    When I send a GET request:    /product-measurement-units/${measurement_unit.m}
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+    And Response body parameter should be:    [data][id]    METR
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot:15]: Please use a variable here accordingly

Comment (gorav-sharma-metacube) [reply tests/api/suite_nonsplit/glue/weight_unit_endpoints/product_measurement_units/positive.robot]: fixed

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Review by Katerynaa-Lavryk — DISMISSED** [review]

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**